stl/inc/yvals_core.h

```diff
@@ -189,6 +189,7 @@
 // P0769R2 shift_left(), shift_right()
 // P0784R7 Library Support For More constexpr Containers
 // P0811R3 midpoint(), lerp()
+// P0849R8 auto(x): decay-copy In The Language

```

Comment:
Is the comment correct given that Core Language part isn't there?

Reply:
I'm in favor of `(library part only)`.

Reply:
With later removing when the Core part is there too to avoid confusion.

Reply:
> * In practice, we'd mostly need to ensure that cppreference is aware of the status

As the one who mentioned the library part of P0849R8 in cppreference, I described the changes as some wording correction ("defect report") against C++20, and said that implementations have been doing the reasonable thing at first (so no DR list was used).😹